lib/cookies.js

```diff
@@ -91,7 +91,7 @@ Cookies.prototype.set = function(name, value, opts) {
 
   if (opts && signed) {
     if (!this.keys) throw new Error('.keys required for signed cookies');
-    cookie.value = this.keys.sign(cookie.toString())
+    cookie.value = this.keys.sign(cookie.toString()).toString('base64')
```

Comment:
I pulled down this module, which installed keygrip 1.0.1 by default, added `console.log(typeof this.keys.sign(cookie.toString()))` and it printed `string`. I also added `console.log(Buffer.isBuffer(this.keys.sign(cookie.toString())))` and it printed `false`. The `.toString('base64')` does nothing, since the `.toString()` on a string just returns the same string, and it doesn't accept any arguments. Your change here can be removed, as it's just doing `'apple'.toString('base64')`, which returns `'apple'`.